Engine/src/Engine/Scene.cpp

```diff
@@ -53,9 +55,9 @@ namespace Engine
 		glDeleteProgram(m_programId);
 	}
 
-	void Scene::onRuntimeUpdate()
```

Comment:
This is just the update function that calls whatever needs updating, correct? So it renders, swap OpenGL buffers, and then polls all events (not sure what events would be included, but that seems more like a comment for that particular function)

Reply:
it's an update function for runtime. It does all those things, but commenting it to that level of detail just creates a lot of maintenance because this function might be modified by other entities and scripting later on.

Reply:
In this case I think descriptive method names are more valuable than a bunch of comments

Reply:
I think in this case, yes, that's true, but perhaps we should at least list what the function should be updating in a comment. Since to me, I feel like this should only update rendering, but if that's the case, I have no idea why poll events is relevant (mostly because I don't know what that function does). 

Thoughts?

Reply:
both glswapbuffers and glfwPollEvents are commented by the library where they're from. It's hard to comment what the function would be updating because this function would be updating anything that needs to be updated. Right now it's only rendering the scene and the subsequent glfw methods, but later on, it might be entity behavior, entity creation/deletion (if we don't have pooling), and so on. How this function works isn't any different than what's already described in the .h

Reply:
Ah, I didn't think about the header comment, my bad.
If so, that should be fine. I'd advise this get a comment if the function becomes more complex, as having people update a few lines when the function is changed should not cause as much overhead as explaining it to someone who doesn't understand it.